unicodetools/data/security/14.0.0/data/source/removals.txt

```diff
@@ -692,6 +692,9 @@ AB63; uncommon-use # LATIN SMALL LETTER UO
 \p{block=Ancient_Greek_Musical_Notation} ; obsolete
 \p{block=Phaistos_Disc} ; obsolete
 
+# 14.0
+\p{block=Znamenny_Musical_Notation} ; technical
```

Comment:
Here is a manual change in an input file. Before this, Znamenny symbols and marks got Identifier_Type=Recommended or Not_XID.